engine/src/scriptpt.cpp

```diff
@@ -1310,33 +1309,55 @@ Parse_stat MCScriptPoint::lookup(Script_point t, const LT *&dlt)
 	return PS_NO_MATCH;
 }
 
-bool MCScriptPoint::lookupconstantvalue(const char *& r_value)
+bool MCScriptPoint::lookupconstantintable(uint2& r_position)
```

Comment:
Since this is wholesale update - just use int rather than uint2 through this (same for cond)... It will result in more efficient code on all platforms.